FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -558,6 +558,128 @@ - (void)testFetchConfigsFailed {
                                }];
 }
 
+- (void)testFetchConfigsFailedErrorNoNetwork {
```

Comment:
nit: I wonder if it is too big effort to break this test to several methods, e.g. refactor all mocks setup to a separate method that can be called from the for-loop (and may be potentially used by other tests). Also, separate methods for configuring individual mocks may improve readability and code reuse. 

Feel free to consider it as a separate improvements.

Reply:
sg. Other tests have the mocks initialized in the setup section. It is only the testFetchConfigsFailed and now this test that need a different response code back. But yes, makes sense to refactor..will make that a separate change.

Reply:
maybe add a todo here.

Reply:
Added.